docs/wg-terminology.md

```diff
@@ -0,0 +1,39 @@
+# Working Group Terminology
+
+Clarifications and additions.
+
+### RDF Triple
+[RDF Triples are defined in RDF 1.2](https://www.w3.org/TR/rdf12-concepts/#section-triples).
+
+Sometimes informally called: "Triple Type", "Abstract Triple".
+
+### Asserted Triple
+
+A triple is asserted with respect to a graph, when it is an element of a graph.
```

Comment:
I am proposing an alternative term here: "Triple Membership". I see two advantages of this term:
1. It can be used as a noun (like it is also possible for terms such as "triple occurrence" and "triple instance"). In other words, it allows us to talk about making statements about a membership of a triple (in some graph), which is not possible for the term "assertion" because the word "assertion" may also be understood as the action of asserting.
2. It does not carry any meaning with respect to semantics. In contrast, the term "asserted triple" already implies something regarding the semantics. (Note that I am not saying that we should revisit the question of whether a triple that is a member/element of a graph is considered as asserted within this graph. I am just saying that the term "membership" allows us to talk about triples in graphs in a semantically neutral way. The property of a triple being asserted in a graph is something that we define based on the property of a triple being a member of a graph.)

Reply:
I think this is a valuable distinction. A graph is defined as a set of triples, so certainly each triple is a member of that set. With that, we should also:

1. State that "When a triple is a member of a graph, the statement it represents is asserted."
3. Define "Asserted Statement" as "The statement is asserted with respect to the meaning of the graph."

Reply:
_Text from RDF 1.2 Concepts_

This hasn't changed from RDF 1.1:

https://www.w3.org/TR/rdf12-concepts/#resources-and-statements
> Asserting an [RDF triple](https://www.w3.org/TR/rdf12-concepts/#dfn-rdf-triple) says that some relationship, indicated by the [predicate](https://www.w3.org/TR/rdf12-concepts/#dfn-predicate), holds between the [resources](https://www.w3.org/TR/rdf12-concepts/#dfn-resource) [denoted](https://www.w3.org/TR/rdf12-concepts/#dfn-denote) by the [subject](https://www.w3.org/TR/rdf12-concepts/#dfn-subject) and [object](https://www.w3.org/TR/rdf12-concepts/#dfn-object).

New text in RDF 1.2 Concepts (added 2023-04-27): https://github.com/w3c/rdf-concepts/commit/1d02fffc5752fb258c3b4e3415fb975b9714a94e

https://www.w3.org/TR/rdf12-concepts/#section-triples:
> An asserted triple is an [RDF triple](https://www.w3.org/TR/rdf12-concepts/#dfn-rdf-triple) that is an element of an [RDF graph](https://www.w3.org/TR/rdf12-concepts/#dfn-rdf-graph).

Reply:
Thank you @afs. Then we should not change the terminology (i.e. keep "Asserted triple" and definition; in fact we should just reference the definition that is in 1.2).

Looking at https://www.w3.org/TR/rdf12-concepts/#entailment:

> An [RDF triple](https://www.w3.org/TR/rdf12-concepts/#dfn-rdf-triple) encodes a [statement](https://www.w3.org/TR/rdf12-concepts/#dfn-rdf-statement)—a simple logical expression, or claim about the world.

and again at https://www.w3.org/TR/rdf12-concepts/#resources-and-statements:

> This statement corresponding to an RDF triple is known as an RDF statement.

I see three aspects to this:
1. Abstract syntax: the triple is an element of the graph.
2. Formal semantics: the triple is asserted.
3. Expression: the meaning of the statement (i.e. the relationship in the universe of discourse) that the asserted triple encodes (within the context of the graph, given a chosen interpretation).

(In suggesting "asserted statement" I conflated 2 and 3. I'm not sure if I differentiate 1 and 2 correctly.)